style.css

```diff
@@ -324,3 +324,52 @@ ul {
   position: absolute;
   margin-top: 308px;
 } */
+
+.contact-me {
+  background-image: url("./portfolio/contac-shape-bg.svg");
+  background-color: #6070ff;
+  background-repeat: no-repeat;
+  background-position: 160px 40px;
+  border-top-left-radius: 60px;
+}
+
```

Comment:
- [ ] You are almost there. Just one step to go 🏁 Please ensure that you follow the figma design as much as possible. Right now, your background has a different color. Kindly make sure that you fix this. I know you can do that 💪 For example, you can do that by simple adding a background color `background-color: #fff` to your `footer`, something like below 🔽

## **_Your design 👇_**
![image](https://user-images.githubusercontent.com/106119105/212906498-4af15ed1-95bf-41cf-8448-57328181991e.png)

## **_FIgma design 👇_**
![image](https://user-images.githubusercontent.com/106119105/212906560-a0893027-fd2f-4638-843d-310eaf11ac1f.png)